src/App.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Calculator from './components/Calculator';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
Please ensure that the linter is not disabled.  A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))). Kindly enable all the disabled linters.

Reply:
- Hello @micheaol ! Thanks for the review, but the only reason I had to use the "disable prefer-stateless-function" linter is because the project wants us to use only **class** components [All components should be class-based](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) but the linters are suggesting to use function component, otherwise they will throw an error!

So what approach can I take in this case?  Thank you in advance for your reply.

Reply:
Hi @myelin0,

Please try to call a constructor in your class component like the code below. I believe that will take the error away.

```javascript
// ./src/components/Calculator.js

import React, { Component } from 'react';

class Calculator extends Component {

  constructor() {
    super();
    this.state = { };
  }
  render() {
    // reference as a js object
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}

export default Calculator; // export Calculator component
```

I hope this helps.

Reply:
Yes, it did. Thank you very much! I tried using constructor and super before, but it gave me the error " Useless constructor "and "Component should be pure function", but adding the state made all the difference (doesn't throw an error anymore).
Thank you again, much appreciated! :+1: